components/Admin/csv-loader.js

```diff
@@ -30,7 +31,7 @@ function validateCsv(
   setError,
   t
 ) {
-  let headerOffSet = noHeader ? 1 : 0;
+  let headerOffSet = noHeader ? 0 : 1;
```

Comment:
- No header should mean: The first row needs to be validated and counted as content (questions, answers) because we do not have a header
- With header: Do not validate the first row and do not color it, because it is defining columns and isn't round (green) nor final round (blue)
I was trying to resolve the firefox csv bug, and was puzzled why my csv file format was incorrect, but am I still not getting something? :D

Reply:
Ahh I see what you mean now. 

I guess it can be interpreted in both ways 🤷.

Lets go with "No Header" with your definition. Now I think it makes more sense than what I originally intended.